insolar/vm.go

```diff
@@ -99,7 +99,39 @@ type LogicCallContext struct {
 	Caller          *Reference // Contract that made the call
 	CallerPrototype *Reference // Prototype (base class) of the caller
 
-	TraceID string // trace mark for Jaegar and friends
+	TraceID      string // trace mark for Jaeger and friends
+	pulseFetcher func(ctx context.Context, pn PulseNumber) (Pulse, error)
```

Comment:
function pointer IS NOT acceptable in LogicCallContext, this struct is transferred over network, we already have means to implement such helpers.